app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -760,39 +878,7 @@ export default class SampleViewV2 extends React.Component {
               }
             </UserContext.Consumer>
           </div>
-          {currentTab === "Report" && (
-            <div className={cs.reportViewContainer}>
-              <div className={cs.reportFilters}>
-                <ReportFilters
-                  backgrounds={backgrounds}
-                  onFilterChanged={this.handleOptionChanged}
-                  onFilterRemoved={this.handleFilterRemoved}
-                  sampleId={sample && sample.id}
-                  selected={selectedOptions}
-                  view={view}
-                />
-              </div>
-              <div className={cs.statsRow}>
-                {this.renderReportInfo()}
-                {!!this.countFilters() && (
-                  <span
-                    className={cs.clearAllFilters}
-                    onClick={this.clearAllFilters}
-                  >
-                    Clear All Filters
-                  </span>
-                )}
-              </div>
-              <div className={cs.reportTable}>
-                <ReportTable
-                  data={filteredReportData}
-                  onTaxonNameClick={this.handleTaxonClick}
-                />
-              </div>
-            </div>
-          )}
-          {currentTab === "Antimicrobial Resistance" &&
-            amrData && <AMRView amr={amrData} />}
+          {this.renderTab()}
```

Comment:
Lets keep the tab check here and call renderReport(). 
I am also considering moving the renderReport into a Report component later on.

Reply:
Done.